crates/evm_attestation/integrity/src/lib.rs

```diff
@@ -0,0 +1,141 @@
+use hdi::prelude::*;
+pub mod entry_types;
+pub use entry_types::*;
+pub mod link_types;
+pub use link_types::*;
+pub mod evm_address_to_signing_offer;
+pub use evm_address_to_signing_offer::*;
+pub mod evm_signing_offer;
+pub use evm_signing_offer::*;
+pub mod name_to_evm_signing_offer;
+pub use name_to_evm_signing_offer::*;
+
+
+#[hdk_extern]
+pub fn genesis_self_check(_data: GenesisSelfCheckData) -> ExternResult<ValidateCallbackResult> {
+    Ok(ValidateCallbackResult::Valid)
+}
+pub fn validate_agent_joining(
+    _agent_pub_key: AgentPubKey,
+    _membrane_proof: &Option<MembraneProof>,
+) -> ExternResult<ValidateCallbackResult> {
+    Ok(ValidateCallbackResult::Valid)
+}
+#[hdk_extern]
+pub fn validate(op: Op) -> ExternResult<ValidateCallbackResult> {
```

Comment:
Would be nice to find a way to generalise this since it's repeated so many times now

Reply:
this is another one we have to swallow.. as every zome has these common validation functions.  trying to separate them from the zomes validation files.. wont work and breaks modularity.